2_base_and_sensor.ipynb

```diff
@@ -14,19 +16,56 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Controlling the move base and captureing sensor information\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Objective</b>\n",
+    "    <p>In this notebook, we will learn how to control the move base of HSR and how to access sensor information.</p>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# 台車を制御する"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Control the move base"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "HSRの台車は速度司令で制御することができます。\n",
     "まず、この章で用いるライブラリをインポートし、初期化をします。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "The move base of HSR can be controlled by velocity command.\n",
```

Comment:
If there are multiple velocity commands, then use the following:

    "The HSR's moving base can be controlled using velocity commands.\n",

If there is just a single velocity command, then use the following:

    "The HSR's moving base can be controlled using the velocity command.\n",